src/edgepi/adc/adc_voltage.py

```diff
@@ -70,17 +69,23 @@ def code_to_voltage(code: list[int], adc_info: ADCReadInfo, calibs: CalibParam)
     Returns:
         `float`: voltage value (V) corresponding to `code`
     """
-    code_bits = bitstring_from_list(code[:adc_info.num_data_bytes])
-    num_bits = adc_info.num_data_bytes * 8
-    code_val = code_bits.uint
 
-    if _is_negative_voltage(code_bits):
+    num_bits = adc_info.num_data_bytes * 8
+    if adc_info.num_data_bytes == ADC1_NUM_DATA_BYTES:
+        code_val = combine_to_uint32(code[0], code[1], code[2], code[3])
```

Comment:
not a big deal, but if you want to avoid repeated args (but less explicit code) you can do this instead:

```python
code_val = combine_to_uint32(*code)
```

Reply:
I'm interested in keeping it explict, so it's implied that elements in `code` are 8 bits, but I didn't know this syntax existed, cool!